tests/actor/test_run_utils.py

```diff
@@ -0,0 +1,112 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Simple tests for the actor runner."""
+
+import time
+from typing import Iterator
+from unittest.mock import patch
+
+import async_solipsism
+import pytest
+import time_machine
+from async_solipsism.socket import asyncio
+
+from frequenz.sdk.actor import actor, run
+
+
+@pytest.fixture(autouse=True)
+def fake_loop() -> Iterator[async_solipsism.EventLoop]:
+    """Replace the loop with one that doesn't interact with the outside world."""
+    loop = async_solipsism.EventLoop()
+    yield loop
+    loop.close()
+
+
+@pytest.fixture
+def fake_time() -> Iterator[time_machine.Coordinates]:
+    """Replace real time with a time machine that doesn't automatically tick."""
+    with time_machine.travel(0, tick=False) as traveller:
+        yield traveller
```

Comment:
Not for this PR, but we probably should move this to a tests/util module.

Reply:
yes, we should. I'll create an issue and post it here once it's done

Reply:
https://github.com/frequenz-floss/frequenz-sdk-python/issues/223